packages/venia-ui/lib/components/Dialog/dialog.js

```diff
@@ -94,7 +108,7 @@ const Dialog = props => {
                             {maybeCloseXButton}
                         </div>
                         <div className={classes.body}>
-                            <div className={classes.contents}>{children}</div>
+                            <div className={classes.contents}>{contents}</div>
```

Comment:
The reason we did not just render `null` when `!isOpen` was because of the desire for a smooth transition to the closed state. Does this maintain that transition or is there a jarring squish of the dialog when the content goes `null`?

Reply:
It does. The smooth transition is done if the whole dialog is unmounted. We are only unmounting the child components.